ui/src/context/MessagesContext.tsx

```diff
@@ -80,6 +80,8 @@ function useDisplayErrorMessage() {
   return context.displayErrorMessage;
 }
 
+// Do we want to keep this?
+// eslint-disable-next-line
 function useDisplaySuccessMessage() {
```

Comment:
what do u think? I think we can actually get rid of it. Success messages are kinda a redundant think anyway